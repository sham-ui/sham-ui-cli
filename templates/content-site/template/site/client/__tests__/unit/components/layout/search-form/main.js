import * as directives from 'sham-ui-directives';
import LayoutSearchFormMain  from '../../../../../src/components/layout/search-form/main.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( LayoutSearchFormMain, {}, {
        directives
    } );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
