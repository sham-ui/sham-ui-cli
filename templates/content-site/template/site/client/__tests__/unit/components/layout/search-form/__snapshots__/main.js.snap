// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "onClose": [Function],
  },
  "html": "
<div class=\\"search\\">
  <div class=\\"container-fluid\\">
    <div class=\\"search-width  text-center\\">
      <button type=\\"button\\" class=\\"close\\">
        <i class=\\"icon-cancel\\"></i>
      </button>
      <form class=\\"search-form\\">
        <input type=\\"search\\" placeholder=\\"What are you looking for?\\" name=\\"query\\">
        <button type=\\"submit\\" class=\\"search-btn\\">search</button>
      </form>
    </div>
  </div>
</div>
",
}
`;
